kahuna/app/controllers/Panda.scala

```diff
@@ -12,11 +14,14 @@ import scala.util.{Success, Try, Failure}
 
 object Panda extends Controller
   with PanDomainAuthActions
+  with ArgoHelpers
   with ArgoErrorResponses {
 
   override lazy val authCallbackBaseUri = Config.rootUri
-  def loginUri: String = Config.loginUri
+  override lazy val loginUriTemplate    = Config.loginUriTemplate
 
+
+  // FIXME: how to use usual Authenticated action helper for this? separate Controller?
```

Comment:
I think it's doable but not directly related to this, so leaving for the next PR.